helpers/deploymentConfig.ts

```diff
@@ -3039,6 +3046,24 @@ export const globalConfig: NetworkConfig = {
             reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
             vTokenReceiver: preconfiguredAddresses.sepolia.VTreasury,
           },
+          {
+            name: "Venus rsETH (Liquid Staked ETH)",
+            asset: "rsETH",
+            symbol: "vrsETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.09", 18),
+            jumpMultiplierPerYear: convertToUnit("3", 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.8", 18),
+            liquidationThreshold: convertToUnit("0.85", 18),
+            reserveFactor: convertToUnit("0.2", 18),
+            initialSupply: "", // to be provided
+            supplyCap: convertToUnit(8_000, 18),
+            borrowCap: convertToUnit(3_600, 18),
+            reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
+            vTokenReceiver: "", // TBD
```

Comment:
On testnet, the received could be the VTreasury

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/413/commits/5d4cbf21469d7a0d5b92e3ddf11c2adf0e257e5b